### ISSUE TITLE
The virus started in America in Summer 2019

### ISSUE THREAD

**Issue by FindtheTruthCOVID19** [Project-Evidence/project-evidence.github.io#94]
1) Vaping illness pneumonia outbreak with exactly exact symptoms started in July 2019, in USA only.
2) Genomic analysis shows that the virus came to Wuhan from somewhere else.

It probably escaped from Fort Detrick in Maryland, during 2019. Fort Detrick researches and specially Sina Bavari as responsible of the program must be investigated.

Check this video for more evidences: https://youtu.be/MSPiUvhWobg

A shorter version of the video can be also found here: https://youtu.be/eCKYNB7fJ0Y

**Comment by abenormle** [comment]
This hypothesis is interesting, but hardly as damning as the authors are suggesting. Raises more questions than it answers, most of which would cover the topic of how this mysterious vaping-associated illness is **not** identical to COVID-19.

Personally, I find issue with the heavy weighting of the "ground glass opacity" findings, because I have learned that these are not especially unique to COVID-19.  I have a child with allergic asthma, and acute flare-ups of this cause the "ground glass opacity" findings on imaging. This actually caused my child to get mis-diagnosed with COVID-19 in the early days of pandemic lockdown.  Later, when we apparently *did actually contract COVID-19*, my asthmatic child had the most alarming respiratory symptoms in our family (as you might expect,) and we had to deal with an initial brush-off from our healthcare provider citing the previous misdiagnosis.

In short, diffuse GGO, appears to be a feature of a number of inflammatory conditions, and the non-malignant ones all appear to involve a type of immune response.
-
The second major issue here is the pattern of cases.  There must be some connection to vaping or a coincidental cultural or environmental factor.  And even if this is a predecessor of SARS-CoV-2, it clearly was not yet contagious in the same manner. Or, perhaps it required some component involved with vaping to trigger sysmptoms? It's a different disease.  

I suppose it could have been a parent of the SARS-CoV-2 that ran wild in Wuhan... But one has to imagine that there was some elaborate mechanism at work (or stranger still, foul play,) that infected U.S. "vapers" and that this led, by happenstance, to traverse the globe, then mutate and trigger an epidemic in the metro area surrounding the Wuhan Institue of Virology, who was famously doing controversial advanced research on the very thing.

That's a lot of coincidence to absorb and believe to be likely, in my opinion.

That isn't to say that there was not some suspicious mystery to the "vaping-associated" illnesses in the US., and the timing is suspicious in and of itself, but the proposed linkage to COVID-19 in Wuhan is lacking... not to say that there is some connection that makes more sense and has yet to be elucidated.

It seems just a credible that some as-yet-to-be-identified vaping ingredient was triggering allergy in susceptible individuals. The lack of government oversight in the US on these vaping products, most of which have imported ingredients, makes is difficult to prevent that sort of thing

-

Lastly, I have to comment on the undeniable air of propaganda in this video and related materials.  From the name "FindtheTruthCOVID19" to the way the material is presented, it makes me feel like I am being "sold" a position, rather than watching an earnest attempt to investigate a hypothesis.  (See the primary document here for a great example of an objective, earnestly investigative approach.)

**Comment by FindtheTruthCOVID19** [comment]
> this mysterious vaping-associated illness is not identical to COVID-19.

Yes it's identical. All the symptoms, CT scans images, lung microscope analysis, treatment, disease evolution and sequels are 100% identical in both diseases.  There is not a single difference between them. 

> I find issue with the heavy weighting of the "ground glass opacity" findings, because I have learned that these are not especially unique to COVID-19. I have a child with allergic asthma, and acute flare-ups of this cause the "ground glass opacity" findings on imaging. This actually caused my child to get mis-diagnosed with COVID-19 in the early days of pandemic lockdown. 

Ground glass opacity is not typical from allergic asthma, so I understand that he was mis-diagnosed with COVID. GGO is one of the characteristics of COVID and EVALI but it can happen on other diseases too. However, these diseases are not identical to COVID and EVALI in the rest, as they are.


> In short, diffuse GGO, appears to be a feature of a number of inflammatory conditions, and the non-malignant ones all appear to involve a type of immune response.

 Again, the question is that GGO is not the only similarity, as I said above they are identical on all its characteristics: Macrophages in the lungs, consolidations,  linear opacities, bronchiestasis, etc


> The second major issue here is the pattern of cases. There must be some connection to vaping or a coincidental cultural or environmental factor. And even if this is a predecessor of SARS-CoV-2, it clearly was not yet contagious in the same manner. Or, perhaps it required some component involved with vaping to trigger sysmptoms? It's a different disease.

There is no need to have any correlation to vaping. You only know the cases of people identified as EVALI because they vaped. The ones that they didn't vape was considered other cause, like flu or just unknown pneumonia, like the cases in France dated in November and December that has been identified recently as COVID. Actually, before the vaping illness outbreak we had in June the case of Fairfax County assisted living facility, a pneumonia outbreak like COVID that affected workers and people living there. Any virus was identified, that was finally said that was  the "common cold" because later they found a common cold virus on some of them, but this is usual on COVID patients too: [https://youtu.be/Q0vZMvMqvHg](https://youtu.be/Q0vZMvMqvHg)

Take a look to these COVID rough numbers:
- Asymptomatic: 80% of all cases
- Hospitalized:  15% of all identified cases (so 15% of 20%, which is 3% of the total)
- Ends on a ventilator: 10% of the hospitalized cases (that means 0,3% of all the cases)

So if you take a look to the ones from EVALI where 97% of cases ended on a ventilator you can deduce that potentially we had more than 300 cases per each EVALI case detected. Most of them asymptomatic, other with not severe flu/cold compatible symptoms and the grave ones, probably on old people were probable considered as pneumonia of unknown origin.


> one has to imagine that there was some elaborate mechanism at work (or stranger still, foul play,) that infected U.S. "vapers" and that this led, by happenstance, to traverse the globe, then mutate and trigger an epidemic in the metro area surrounding the Wuhan Institue of Virology, who was famously doing controversial advanced research on the very thing.

The other evidences on the videos refers to the fact that it did NOT started in Wuhan. The virus was identified in Wuhan and the outbreak has explode from there but the virus has been identified before in France, Italy and recently in Spain in waste waters from March 2019. Also the genome analysis demonstrate the it didn't begin in Wuhan. Wuhan Institute of Virology investigated bat virus and coronavirus, as well as Fort Detrick in Maryland and many other researches. Actually Wuhan Institute was working for EEUU. From what I know there is not any evidence that link COVID to  Wuhan Institute of Virology right? In the other hand, there are scientific evidences that demonstrate that the virus existed out of Wuhan before the outbreak official starting date. 


> That's a lot of coincidence to absorb and believe to be likely, in my opinion.

This theory is the only one that I know that fits with all the scientific findings to date. Any other theory goes against the facts.

> the proposed linkage to COVID-19 in Wuhan is lacking... not to say that there is some connection that makes more sense and has yet to be elucidated.

Well, the objective f this project is to find the truth. This may be part o the truth but as you say we need more. Let's work together to find the truth, but in the good direction, the direction of the facts and scientific evidence, not politics or conspiracy theories. 

The US must investigate COVID19 origin in their territory, do test of 2019 waste water samples like Italy and Spain, re-analyze CT scans and samples of patients with pneumonia like France and of course test vaping illness samples from COVID19. With than information we would be able to fill the gaps on this theory or ,hopefully, debunk it.

> It seems just a credible that some as-yet-to-be-identified vaping ingredient was triggering allergy in susceptible individuals. 

Credible? This is not a matter of being credible or not, it's a matter of medical research. At the date of today any contaminant has been found on the vaping products yet as cause of the illness. The official CDC position accusing Vitamin E only because they found it on the lung patients is ridiculous. Also, not all the affected vaped THC, or the same brands, and that possibility has been investigated a lot. In Europe they don't control vape products more than US, they are the same. In any case, if we apply the argument of being credible or not it would be an incredible coincidence that a contaminant on a vaping product would cause 100% exact the same symptoms than COVID and just started to be used a couple of months before the COVID outbreak started.

> Lastly, I have to comment on the undeniable air of propaganda in this video and related materials. From the name "FindtheTruthCOVID19" to the way the material is presented, it makes me feel like I am being "sold" a position, rather than watching an earnest attempt to investigate a hypothesis. (See the primary document here for a great example of an objective, earnestly investigative approach.)

This video pretends to show a list of evidences but in any way pretends to "enforce" a specific theory, so any new information in other direction will be welcome and very helpful to clarify the truth about COVID-19 origin. I'll summarize the evidences in the video that in my opinion should guide further investigations on that direction:

1. The virus did not started in Wuhan. Proofs:
2. The virus was circulating in EEUU during summer 2019.
3. Athletes from many countries reported COVID symptoms in Wuhan.
4. After the Military Games the virus was spread around  the world.
5. Fort Detrick investigated bat viruses and coronavirus treatments and was closed in summer 2019 because security reason.
